### DISCUSSION TITLE
.json animation on local dev visible but not on vercel deployment

### DISCUSSION THREAD

**Discussion by GeBon22** [neuefische/questions#346] [Web] [answered]
As I added several animations on my landing page, I pushed my latest code changes and deployed the newest version on vercel -> it shows the animation <svg> Tag including its contents, but it is not rendering the animation itself. 
What I tried: 
I added a new folder for animations using public as root folder and changed the respective pathing following the solutions suggested in [this Thread](https://github.com/orgs/neuefische/discussions/324).
I turned off Ghostery with no visible effect as it may sometimes block animations.

![grafik](https://user-images.githubusercontent.com/114496506/204806016-8187b5b4-43a5-4276-8021-d54dbd969801.png)

[Here ](https://capstone-project-f77sr40wi-gebon22.vercel.app/)you can find the deployed branch on vercel.

[Here ](https://github.com/GeBon22/Capstone-Project/pull/9#issue-1469388697)you can find my PR.

**Comment by actyralf** [comment]
I think you have to drop the `../public` part of your json-URL. Next.js treats the public folder as the root folder: https://nextjs.org/docs/basic-features/static-file-serving

**Reply by GeBon22** [reply]
Thats not working for me sadly.. When changing the path name to `../WiderrufAnimation` or `/WiderrufAnimation` its showing an error and no animation.

**Reply by actyralf** [reply]
Hey Georg,

after stripping out line for line and file for file to reduce your app to the bare minimum I think I found the solution: It seems that this is a problem with Next.js version 13.0.4. Please upgrade your Next.js version by running `npm install next@13.0.5` and try to deploy again.

**Comment by actyralf** [comment, marked as answer]
Hey Georg,

after stripping out line for line and file for file to reduce your app to the bare minimum I think I found the solution: It seems that this is a problem with Next.js version 13.0.4. Please upgrade your Next.js version by running `npm install next@13.0.5` and try to deploy again.

**Reply by GeBon22** [reply]
this is it - Thanks a lot
